File: Examples/VisualBasic/Rendering-Printing/LoadHyphenationDictionaryForLanguage.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.IO
Imports System.Drawing
Imports Aspose.Words
Imports Aspose.Words.Layout
Imports Aspose.Words.Rendering
Public Class LoadHyphenationDictionaryForLanguage
    Public Shared Sub Run()
        ' ExStart:LoadHyphenationDictionaryForLan
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_RenderingAndPrinting()

        ' Load the documents which store the shapes we want to render.
        Dim doc As New Document(dataDir & Convert.ToString("TestFile RenderShape.doc"))
        Dim stream As Stream = File.OpenRead(dataDir & Convert.ToString("hyph_de_CH.dic"))
        Hyphenation.RegisterDictionary("de-CH", stream)

        dataDir = dataDir & Convert.ToString("LoadHyphenationDictionaryForLanguage_out_.pdf")
        doc.Save(dataDir)
        ' ExEnd:LoadHyphenationDictionaryForLan
        Console.WriteLine(Convert.ToString(vbLf & "Hyphenation dictionary for special language loaded successfully." & vbLf & "File saved at ") & dataDir)
    End Sub
End Class

File: Examples/CSharp/Rendering-Printing/LoadHyphenationDictionaryForLanguage.cs

using System;
using System.IO;
using System.Drawing;

using Aspose.Words;
using Aspose.Words.Layout;
using Aspose.Words.Rendering;
using Aspose.Words.Saving;
using Aspose.Words.Drawing;
using System.Drawing.Imaging;
using Aspose.Words.Tables;

namespace Aspose.Words.Examples.CSharp.Rendering_and_Printing
{
    class LoadHyphenationDictionaryForLanguage
    {
        public static void Run()
        {
            //ExStart:LoadHyphenationDictionaryForLanguage
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_RenderingAndPrinting(); 

            // Load the documents which store the shapes we want to render.
            Document doc = new Document(dataDir + "TestFile RenderShape.doc");
            Stream stream = File.OpenRead(dataDir + @"hyph_de_CH.dic");
            Hyphenation.RegisterDictionary("de-CH", stream);

            dataDir = dataDir + "LoadHyphenationDictionaryForLanguage_out_.pdf";
            doc.Save(dataDir);
            //ExEnd:LoadHyphenationDictionaryForLanguage
            Console.WriteLine("\nHyphenation dictionary for special language loaded successfully.\nFile saved at " + dataDir);           
        }
        
    }
}
